### PR TITLE
Redesign default tab

### PR DESCRIPTION
**Pull request by RStankov**
### Scope

First step of implementing the new default tab UI.

screen shot 2017-01-20 at 19 54 15

### CODE REVIEW

File: src/apps/tabs/components/PostGroup.react.js [hunk 1]
```diff
@@ -16,49 +16,31 @@ let groupByDay = require('../util/groupByDay');
...
-          {posts}
-        </div>
-      );
-    });
+export default function PostGroup({ posts }) {
+  let groups = groupByDay(posts);
+  let out = Object.keys(groups).map(function(day) {
+    let date = moment(new Date(day));
+    let humanDay = getDay(date);
+    let monthDay = date.format('MMMM Do');
```
Comment (andreasklinger) [inline src/apps/tabs/components/PostGroup.react.js]: ;)

Comment (andreasklinger) [inline src/apps/tabs/components/PostGroup.react.js]: should make linters more aggressive

**Review by andreasklinger — APPROVED** [review]

File: src/apps/tabs/components/DefaultTab.react.js [hunk 2]
```diff
@@ -18,10 +18,16 @@ let api = require('../../../common/api');
 const CACHE_KEY = process.env.PRODUCTS_CACHE_KEY;
 
 /**
+ * Debuger.
```
Comment (vesln) [inline src/apps/tabs/components/DefaultTab.react.js]: this is not a debugger

File: src/apps/tabs/components/DefaultTab.react.js [hunk 3]
```diff
@@ -43,18 +49,20 @@ let fetch = async.queue(function(daysAgo, cb) {
...
    * Return initial state.
-   *
-   * @returns {Object}
    */
 
-  getInitialState() {
+  constructor(props) {
+    super(props);
+
+    this.loadNext = this.loadNext.bind(this);
```
Comment (vesln) [inline src/apps/tabs/components/DefaultTab.react.js]: `loadNext = () => {}`?

Comment (RStankov) [inline src/apps/tabs/components/DefaultTab.react.js]: This is not enabled in babel, yet.

File: src/apps/tabs/components/PostGroup.react.js (hunk 1 above)
Comment (RStankov) [inline src/apps/tabs/components/PostGroup.react.js]: we actually don't have any linters here :D

**Comment by RStankov** [comment]
Merged.

**Pull request merged**